examples/multiple_data_source_examples/public/application.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { DataSourcePluginSetup } from 'src/plugins/data_source/public';
```

Comment:
Should we use the relative path for the import?

Reply:
We don't need to, similar here https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/examples/search_examples/server/routes/server_search_route.ts#L31